airbyte-scheduler/persistence/src/main/resources/job_history_purge.sql

```diff
@@ -18,8 +18,8 @@ WHERE
             ) counts ON
             jobs.scope = counts.scope
         WHERE
-            -- job must be at least MINIMUM_AGE_IN_DAYS old or connection has more than EXCESSIVE_NUMBER_OF_JOBS 
-(
+            /* job must be at least MINIMUM_AGE_IN_DAYS old or connection has more than EXCESSIVE_NUMBER_OF_JOBS */
```

Comment:
I had to manually change these comments to this type before running the new Spotless version. Otherwise it puts the parenthesis on the same line as the comment.